src/i18n/pt-br.js

```diff
@@ -0,0 +1,445 @@
+        walletLogLevel: "NIVEL DE LOG DA CARTEIRA",
+        walletName: "NOME DA CARTEIRA",
+        walletRPCPort: "PORTA RPC DACARTEIRA",
+        walletStoragePath: "DIRECTORIA DE SALVAMENTO DA CARTEIRA",
+
+        // These are specific labels which do not get uppercased
+        confirmNewPassword: "Confirmar Nova Palavra-Passe",
+        newPassword: "Nova Palavra-Passe",
+        oldPassword: "Antiga Palavra-Passe",
+        rescanFullBlockchain: "Re-examinar o blockchain completo",
+        rescanSpentOutputs: "Re-examinar saídas gastas",
+        transactionNotes: "Notas de Transações",
+        chooseNetwork: "Seleccione uma Rede",
+        network: "Rede"
+    },
+    footer: {
+        ready: "Pronto",
+        scanning: "Examinando",
+        status: "Estado",
+        syncing: "SINCRONIZANDO",
+        remote: "Remoto",
+        wallet: "Carteira"
+    },
+    menuItems: {
+        about: "Sobre",
+        changePassword: "Alterar Palavra-Passe",
+        copyAddress: "Copiar Endereço",
+        copyQR: "Copiar código QR",
+        copySeedWords: "Copiar palavras semente",
+        copySpendKey: "Copiar chave de gasto",
+        copyTransactionId: "Copira ID da transação",
+        copyViewKey: "Copiar chave de visualização",
+        createNewWallet: "Criar nova carteira",
+        deleteWallet: "Apagar carteira",
+        exit: "Saír da Carteira GUI Loki",
+        importOldGUIWallet: "Importar carteiras da antiga carteira GUI",
+        manageKeyImages: "Administrar Imágens de Chave",
+        openWallet: "Abrir carteira",
+        rescanWallet: "Re-examinar Carteira",
+        restoreWalletFile: "Restaurar carteira a partir de ficheiro",
+        restoreWalletSeed: "Restaurar carteira a partir da semente",
+        saveQR: "Salvar código QR para um ficheiro",
+        sendToThisAddress: "Enviar para este endereço",
+        settings: "Configurações",
+        showDetails: "Show details",
+        showPrivateKeys: "Mostrar Chaves Privadas",
+        showQRCode: "Mostrar Código QR",
+        switchWallet: "Mudar de Carteira",
+        viewOnExplorer: "Visualizar no explorador"
+    },
+    notification: {
+        positive: {
+            addressCopied: "Copiar endereço para o clipboard",
+            bannedPeer: "{host} banido até {time}",
+            copied: "{item} copiado para o clipboard",
+            itemSaved: "{item} salvado em {filename}",
+            keyImages: {
+                exported: "Imágens de Chave exportadas para {filename}",
+                imported: "Imágens de Chave importadas"
+            },
+            passwordUpdated: "Palavra-Passe actualizada",
+            qrCopied: "Código QR ccopiado para o clipboard",
+            registerServiceNodeSuccess: "Nódulo de serviço registado com sucesso",
+            sendSuccess: "Transação enviado com sucesso.",
+            stakeSuccess: "Participação com sucesso",
+            transactionNotesSaved: "Nota de transação salvada."
```

Comment:
transactionNotesSaved: "Nota de transação salva."

Reply:
The "participativo regular" is preferred in the active voice. In the passive I would agree "A nota de transação foi salva" - but here we in the active voice.
https://duvidas.dicio.com.br/salvado-ou-salvo/

Reply:
I actually prefer "A nota de transação foi salva" - what do you think?

Reply:
perfect. Yeah i also messed up in the plural on that one. Salvada was not sounding nice to me at all :P